Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -452,9 +451,15 @@ end
 function PowerGenerator_UpdateAction(furniture, deltatime)
     if (furniture.JobCount() < 1 and furniture.Parameters["burnTime"].ToFloat() == 0) then
         furniture.PowerValue = 0
+<<<<<<< 6d42db4eadcaee2c1ac85d403c9c06703997b463
```

Comment:
You have incorrectly merged this file and left the conflict notes in, preventing the file from parsing.